Ginger/GingerCoreNET/GeneralLib/EmailReadGmailOperations.cs

```diff
@@ -40,7 +41,7 @@ public Task ReadEmails(EmailReadFilters filters, EmailReadConfig config, Action<
             ImapClient client = new ImapClient();
             try
             {
-                client.Connect("imap.gmail.com", 993, true);
+                client.Connect(config.IMapHost, Int32.Parse(config.IMapPort), true);
```

Comment:
Please add validation if Parse of IMapPort fails in case of invalid value in IMapPort